ecl/data_source_ecl_network_internet_service_v2_test.go

```diff
@@ -80,11 +80,18 @@ func testAccCheckNetworkV2InternetServiceDataSourceID(n string) resource.TestChe
 }
 
 func testAccReturnMinimalSubmaskLength(region string) int {
-	minimal_submask_length := 26
-	if region == "lab4ec" {
+	var minimal_submask_length int
```

Comment:
I think CamelCase is better for var name.

Reply:
You are right!
I fixed that point. I'd like you to check this again.